database/inflections/to_csv.mjs

```diff
@@ -0,0 +1,88 @@
+import { stdin as input, argv } from 'node:process'
+import { createInterface } from 'node:readline'
+
+const parts_of_speech = [
+	'Adjective',
+	'Adverb',
+	'Noun',
+	'Verb',
```

Comment:
I just realized that it's possible for a language project to have unique forms for parts-of-speech other than these 4. I'm not aware of any of our current projects that do, but it's possible. So we can always address that as it's needed - for now these 4 should suffice.

Reply:
let's address this in #8